src/track/track.h

```diff
@@ -436,6 +436,7 @@ class Track : public QObject {
     // adjusted in the opposite direction to compensate (no audible change).
     void replayGainAdjusted(const mixxx::ReplayGain&);
     void colorUpdated(const mixxx::RgbColor::optional_t& color);
+    void ratingUpdated(int rating);
```

Comment:
I would prefer the neutral and agnostic _changed_ terminology. That would be inconsistent with the existing signals, but we could add a TODO to those later.